src/main/java/com/agileactors/usersproject/controllers/UsersController.java

```diff
@@ -11,6 +11,10 @@
 @RestController
 @RequestMapping("api/users")
 public class UsersController {
+    public UsersController(UsersRepository usersRepository) {
```

Comment:
1. Use Autowired in constructor. 2 . Remove dependency with repository and use service instead